cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/CreateController.java

```diff
@@ -83,7 +84,7 @@ public void run() {
             try {
                 yamlFileContent = FileUtils.readLocalFile(commandOptions.getYamlFilePath());
             } catch (IOException e) {
-                System.err.println(e.getMessage());
+                Log.exception(e, "Failed to read YAML file");
```

Comment:
Do we want to show the user the Stacktrace in such cases or just a description of the error?

Reply:
We could just show the description. I opted not to do so for two reasons. First, our user base are presumably developers who can handle stack traces, and probably are grateful to see all the details. Second, in CI environments, you can't really debug problems interactively, and a stacktrace is already a great source of information when things go wrong.

Reply:
I also think it is good, because our users are mostly devs and have also thought about it. :) 
I think the best would be to have a trade off. So in my opinion we should also write the logs in an additional textfile with all stacktraces, where the user (mostly the devs) can check it for non obvious reasons/errors.

Reply:
Another viable approach is to introduce a second environment variable and have the `exception` method log the exception _only_ if that one is set. This could also be extended to logging them to a file always and on the commandline only if the variable is set.

However, logging them always is not a big deal either (we only log them when there's fatal errors leading to the termination of the program), and checking every command for an error and if it crashed, cat a file, that's highly annoying to program in a shell script.

Reply:
Your suggestion sounds good. :) I would leave it for now as I am the same opinion as you!